pkg/reconciler/reconcile.go

```diff
@@ -19,5 +34,12 @@ func Reconcile(logger *zap.Logger, client *db.Database, req forman.Request) form
 	}
 	logger.Info("Provider", zap.String("clusterId", provider.ClusterID))
 
-	return forman.Result{}
+	result, err := r.Reconcile(ctx)
```

Comment:
```suggestion
	r := FleetReconciler{
		ctx: ctx,
	}
	result, err := r.Reconcile()
```

Reply:
Ack, however we are attaching the supplied ctx at the start of Reconcile method.

Reply:
done.